website/docs/r/appautoscaling_policy.html.markdown

```diff
@@ -89,6 +89,37 @@ resource "aws_ecs_service" "ecs_service" {
 }
 ```
 
+### Aurora Read Replica Autoscaling
+
+```hcl
+resource "aws_appautoscaling_target" "replicas" {
+  service_namespace  = "rds"
+  scalable_dimension = "rds:cluster:ReadReplicaCount"
+  resource_id        = "cluster:aurora-cluster-id"
+  min_capacity       = 1
+  max_capacity       = 15
+}
+
+resource "aws_appautoscaling_policy" "replicas" {
+  name               = "cpu-auto-scaling"
+  service_namespace  = "rds"
+  scalable_dimension = "rds:cluster:ReadReplicaCount"
+  resource_id        = "cluster:aurora-cluster-id"
```

Comment:
Nitpick: we can point to the target directly to get rid of the `depends_on`: `"${aws_appautoscaling_target.replicas.resource_id}"`